extensions/resource-deployment/src/services/tools/azdataTool.ts

```diff
@@ -20,11 +21,11 @@ export class AzdataTool extends ToolBase {
 	}
 
 	get name(): string {
-		return 'azdata';
+		return AzdataToolName;
 	}
 
 	get description(): string {
-		return localize('resourceDeployment.AzdataDescription', "A command-line utility written in Python that enables cluster administrators to bootstrap and manage the Big Data Cluster via REST APIs");
+		return localize('resourceDeployment.AzdataDescription', "A command-line utility to bootstrap and manage Big Data Clusters and ARC clusters via REST APIs");
```

Comment:
please confirm the string change with Dinakar and Mihaela.

Reply:
I have sent an email to them. If the response gets delayed, I will pul the change out from this PR to its own PR.

Reply:
reverting from this PR and will do its own PR in the future.